### DISCUSSION TITLE
Wrong billing information despite correct name (possible bug)

### DISCUSSION THREAD

**Discussion by r0neko** [community/community#78631] [GitHub Education]
### Hi everyone,

Hello. I've been trying to register for GitHub Education and I've been having some sort of bug in my opinion. After uploading my proof of being a student (more exactly a receipt from the bursier,) I get prompted to update my billing information according to this message:

**Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.**

I did that, and I made sure to provide accurate informations (including diacritics,) but GitHub still is not allowing me to register.

What can I do in this case in order to apply?
Thanks in advance.

**Comment by ghost** [comment]
hello

**Comment by ghost** [comment]
sssgg

**Comment by Hilmican66** [comment]
Bu Git Hup Ne işe yarıyor Mit Falan Diyo ve yazılımla Alakalı çıkıyor Bunun İçin bana Bilgi Verirmisiniz

**Reply by Hilmican66** [reply]
Yazilimlarim neden Git hup Bağlı

**Comment by Patrick0482** [comment]
I have the same problem

**Comment by r0neko** [comment]
I got another receipt today from the university and I managed to register. I'll consider this closed.
